Makefile

```
@@ -28,6 +28,11 @@ test:
 test/update-snapshots:
 	@TZ=UTC UPDATE_SNAPSHOTS=true go test ./...
 
+.PHONY: test/robustness
+test/robustness:
+	@cd integration/subject && npm install --include=dev
+	@RESULTS=`find . -name "README.md" | grep -v "\/\." | xargs dirname | uniq | xargs -n1 ./runme json --chdir`
```

Or this to be more explicit with the intention:

```suggestion
	find . -name "README.md" | grep -v "\/\." | xargs dirname | uniq | xargs -n1 ./runme json --chdir &>/dev/null
```

---

Good call. Also wouldn't want whatever buffer's behind `RESULTS` to run into an overflow (if that's possible) 🤷 .